### PR TITLE
add armRoleReceiver to ActionGroup API and update API version to 2018-09-01

### PR DESCRIPTION
**Pull request by RyanD1**
and update API version to 2018-09-01

This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/monitor/resource-manager/microsoft.insights/stable/2018-09-01/actionGroups_API.json [hunk 1]
```diff
@@ -0,0 +1,832 @@
...
+                    "description": "The list of logic app receivers that are part of this action group."
+                },
+                "azureFunctionReceivers": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/AzureFunctionReceiver"
+                    },
+                    "description": "The list of azure function receivers that are part of this action group."
+                },
+                "armRoleReceivers": {
```
Comment (RyanD1) [inline specification/monitor/resource-manager/microsoft.insights/stable/2018-09-01/actionGroups_API.json]: change

File: specification/monitor/resource-manager/microsoft.insights/stable/2018-09-01/actionGroups_API.json [hunk 2]
```diff
@@ -0,0 +1,832 @@
...
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/ArmRoleReceiver"
+                    },
+                    "description": "The list of arm role receivers that are part of this action group."
+                }
```
Comment (RyanD1) [inline specification/monitor/resource-manager/microsoft.insights/stable/2018-09-01/actionGroups_API.json]: Change: adding armRoleReceiver

File: specification/monitor/resource-manager/microsoft.insights/stable/2018-09-01/actionGroups_API.json [hunk 3]
```diff
@@ -0,0 +1,832 @@
...
+                "roleId": {
+                    "type": "string",
+                    "description": "The arm role id."
+                }
+            },
+            "required": [
+                "name",
+                "roleId"
+            ]
+        },
```
Comment (RyanD1) [inline specification/monitor/resource-manager/microsoft.insights/stable/2018-09-01/actionGroups_API.json]: Change: adding armRoleReceiver

File: specification/monitor/resource-manager/microsoft.insights/stable/2018-09-01/examples/createOrUpdateActionGroup.json [hunk 4]
```diff
@@ -0,0 +1,297 @@
...
+                        "functionName": "HttpTriggerCSharp1",
+                        "httpTriggerUrl": "https://testfunctionapp.azurewebsites.net/api/HttpTriggerCSharp1?code=4CopFfiXqUQC8dvIM7F53J7tIU3Gy9QQIG/vKAXMe2avhHqK3/jVYw=="
+                    }
+                ],
+                "armRoleReceivers": [
+                    {
+                        "name": "Sample armRole",
+                        "roleId": "8e3af657-a8ff-443c-a75c-2fe8c4bcb635"
+                    }
+                ]
```
Comment (RyanD1) [inline specification/monitor/resource-manager/microsoft.insights/stable/2018-09-01/examples/createOrUpdateActionGroup.json]: Change: adding armRoleReceiver

File: specification/monitor/resource-manager/microsoft.insights/stable/2018-09-01/examples/createOrUpdateActionGroup.json [hunk 5]
```diff
@@ -0,0 +1,297 @@
...
+                            "functionName": "HttpTriggerCSharp1",
+                            "httpTriggerUrl": "https://testfunctionapp.azurewebsites.net/api/HttpTriggerCSharp1?code=4CopFfiXqUQC8dvIM7F53J7tIU3Gy9QQIG/vKAXMe2avhHqK3/jVYw=="
+                        }
+                    ],
+                    "armRoleReceivers": [
+                        {
+                            "name": "Sample armRole",
+                            "roleId": "8e3af657-a8ff-443c-a75c-2fe8c4bcb635"
+                        }
+                    ]
```
Comment (RyanD1) [inline specification/monitor/resource-manager/microsoft.insights/stable/2018-09-01/examples/createOrUpdateActionGroup.json]: Change: adding armRoleReceiver

File: specification/monitor/resource-manager/microsoft.insights/stable/2018-09-01/examples/createOrUpdateActionGroup.json [hunk 6]
```diff
@@ -0,0 +1,297 @@
...
+                            "functionAppResourceId": "/subscriptions/5def922a-3ed4-49c1-b9fd-05ec533819a3/resourceGroups/aznsTest/providers/Microsoft.Web/sites/testFunctionApp",
+                            "functionName": "HttpTriggerCSharp1",
+                            "httpTriggerUrl": "https://testfunctionapp.azurewebsites.net/api/HttpTriggerCSharp1?code=4CopFfiXqUQC8dvIM7F53J7tIU3Gy9QQIG/vKAXMe2avhHqK3/jVYw=="
+                        }
+                    ],
+                    "armRoleReceivers": [
+                        {
+                            "name": "Sample armRole",
+                            "roleId": "8e3af657-a8ff-443c-a75c-2fe8c4bcb635"
+                        }
```
Comment (RyanD1) [inline specification/monitor/resource-manager/microsoft.insights/stable/2018-09-01/examples/createOrUpdateActionGroup.json]: Change: adding armRoleReceiver

**Comment by RyanD1** [comment]
@ravbhatnagar Could you help review?
Mostly everything is copied directly from 2018-03-01 API-version which was already approved. We are not introducing a new API, but a new receiver type and updating the API version so the PR may seem falsely large. I've marked all the changes actually made (diff between 2018-03-01 and 2018-09-01). Please let me know if you have any questions, thanks!

**Comment by ravbhatnagar** [comment]
This PR has added the armroleReceiver type only. PLease update the description to clarify that these are the Azure RBAC roles and only built-in are currently supported. Signing off.

**Review by sarangan12 — APPROVED** [review]

**Pull request merged**